featuretools/tests/primitive_tests/test_postal_primitives.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+import pandas as pd
+from woodwork.accessor_utils import _is_spark_series
+
+from featuretools.primitives.standard.transform.postal import (
+    OneDigitPostalCode,
+    TwoDigitPostalCode,
+)
+
+
+def one_digit_postal_code_test(postal_series):
```

Comment:
What are we doing in this test and `two_digit_postal_code_test`? There are no assertions?

Reply:
Those are helper functions I wrote to simplify the logic in the primitive tests. They return the actual and expected values to be compared in the tests functions. I would be happy to move that logic back to the test functions if it would be cleaner.

Reply:
Oh, I see now. I missed that before, obviously.

I think it is fine to use helper functions, but how about this:
- Rename them to something that doesn't include the `test` terminology. Reading the function name made me think we were testing something inside the function.
- Try to combine these into a single function, maybe pass in the primitive as a parameter instead of hard-coding the primitive name inside the helper function.
- I'm a little concerned that we are manipulating the output of the primitive function before we do our assertions. Is there a way we can update so that the expected values are *exactly* what we expect from the primitive function? That makes it much more clear what we are actually asserting, in my opinion.

Reply:
I refactored the primitive to return `pd.NA` in the case of encountering a null value. I then just convert the Dask or Pyspark objects to a Pandas series, and used the Pandas testing utility. It is cleaner now because we don't have to deal separately with the case of `pd.NA` (if we use `np.testing` methods, we need to convert `pd.NA` to `np.nan`)